packages/azure-services/src/azure-queue/queue.ts

```diff
@@ -71,10 +74,14 @@ export class Queue {
     }
 
     public async createMessage(queue: string, message: unknown): Promise<boolean> {
-        const queueURL = await this.getQueueURL(queue);
-        await this.ensureQueueExists(queueURL);
-
-        return this.createQueueMessage(queueURL, message);
+        return this.retryHelper.executeWithRetries(
```

Comment:
did you intend  have this as `tryCreateMessage` instead? then return false on error.